blocks/init/src/Blocks/components/icon/components/icon-options.js

```diff
@@ -1,9 +1,32 @@
 import React from 'react'; // eslint-disable-line no-unused-vars
 import { __, sprintf } from '@wordpress/i18n';
-import { SelectControl, ToggleControl } from '@wordpress/components';
+import { ToggleControl } from '@wordpress/components';
 import { checkAttr } from '@eightshift/frontend-libs/scripts/helpers';
+import { CustomSelect } from '@eightshift/frontend-libs/scripts/components';
+import { components } from 'react-select';
+
 import manifest from './../manifest.json';
 
+const { icons } = manifest;
+
+const IconPickerOption = props => (
+	<components.Option {...props}>
+		<div className={'icon-option-row'}>
+			<i dangerouslySetInnerHTML={{ __html: icons[props.value] }}></i>
```

Comment:
Does this has to be `dangerouslySetInnerHTML`?

Reply:
If there's any better way to get a (non-JSX) SVG in string form to render inside a React component, then there is a better way (and I'll gladly change it), otherwise we need to use this 😄